.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
From my point of view, VSCode settings should not go into the repository.

Reply:
This is where Visual Studio put the file by default.   I don't use VS so I have no idea what the best practices are.  This was just the result of using VS on this project.

Reply:
Yeah, they are put into the hidden folder in you local working copy. However, you should not add it to the commit list. Rather you should add it to gitignore.

Reply:
In that case, I agree remove (and potentially add to `.gitignore`. I was assuming you felt strongly about including it, I wasn't keen but not enough to reject.